src/software/jetson_nano/services/network/network.cpp

```diff
@@ -69,13 +69,6 @@ void NetworkService::primitiveSetCallback(TbotsProto::PrimitiveSet input)
     {
         primitive_set_msg = input;
     }
-
-    float primitive_set_loss_rate = primitive_tracker.getLossRate();
-    if (primitive_set_loss_rate > PROTO_LOSS_WARNING_THRESHOLD)
-    {
-        LOG(WARNING) << "Primitive set loss rate is " << primitive_set_loss_rate * 100
-                     << "%";
-    }
```

Comment:
should we remove this log altogether? Or increase the warning threshold?

Reply:
The value is included in the RobotStatus, so I dont see any reason to keep it. Currently it just spams everytime we estop and un-estop

Reply:
okay you should delete `PROTO_LOSS_WARNING_TRESHOLD` from the header then

Reply:
bump

Reply:
Missed this comment earlier. Updated